macros/cross_db_utils/width_bucket.sql

```diff
@@ -0,0 +1,29 @@
+{% macro width_bucket(expr, min_value, max_value, num_buckets) %}
+  {{ adapter_macro('dbt_utils.width_bucket', expr, min_value, max_value, num_buckets) }}
+{% endmacro %}
+
+
+{% macro default__width_bucket(expr, min_value, max_value, num_buckets) -%}
+
+    {%- set bucket_width = '((' ~ max_value ~ ' - ' ~ min_value ~ ')/' ~ num_buckets ~ ')' -%}
```

Comment:
I like writing these expressions in a `set` block:
```
{% set bucket_width -%}
  (( {{ max_value }} - {{ min_value }} ) / {{ num_buckets }} )
{% endset %}
```

Reply:
Makes sense, changes pushed!